### DISCUSSION TITLE
create-o7-app's Documentation?

### DISCUSSION THREAD

**Discussion by vipoooooo** [ottomated/create-o7-app#13] [General]
i was browsing to get to know more about create-o7-app but there's none except this repo(or maybe there're some out there that i haven't found). anyway, i hope there a documentation about this like create t3 stack.

**Comment by tom-huntington** [comment]
Otto said you should look at the tanstack query [docs](https://tanstack.com/query/v4/docs/react/guides/ssr#using-hydration)
> Prefetch the query on the server, dehydrate the cache and rehydrate it on the client

as well as the README.md of his adaptor [trpc-svelte-query](https://github.com/ottomated/trpc-svelte-query).

Also these repos are probably using create-o7-app. You can use them as samples https://github.com/ottomated/trpc-svelte-query/network/dependents
